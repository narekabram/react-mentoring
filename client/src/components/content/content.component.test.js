import React from 'react';
import { shallow } from 'enzyme';
import Content from './content';

describe('Content', () => {
    it('Adds Content snapshot testing', () => {
        const component = shallow(<Content/>);

        expect(component).toMatchSnapshot();
    });
});
