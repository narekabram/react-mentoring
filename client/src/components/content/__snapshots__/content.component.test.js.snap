// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content Adds Content snapshot testing 1`] = `ShallowWrapper {}`;
